docs/guides/update-guide/elasticsearch/7-to-8.md

```diff
@@ -0,0 +1,19 @@
+---
+id: 7-to-8
+title: Update Elasticsearch 7 to 8
+description: "Review which adjustments must be made to migrate from Elasticsearch 7 to Elasticsearch 8."
+---
+
+The Camunda Platform is not compatible with the Elasticsearch 8 Curator. This Curator is used to manage
+[data retention](../../../components/concepts/data-retention.md) within Elasticsearch. To upgrade to Elasticsearch 8,
+the Curator must be replaced with
+[Index Lifecycle Management](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html)
+(ILM). Once the upgrade to Elasticsearch 8 is complete, the Curator can be disabled.
```

Comment:
```suggestion
(ILM). Once the update to Elasticsearch 8 is complete, the Curator can be disabled.
```

❓ Can the Curator be disabled after the update? Perhaps it should be disabled before 🤔 I'm unsure.

Reply:
I'd argue after is better. If you do it before the indexes that exist are not configured for ILM yet. So these indexes will never be cleaned.

Reply:
That's a good point, we should clarify this in these docs